naucse/static/css/pygments-lovelace-style.css

```diff
@@ -68,3 +68,4 @@
 .highlight .vi { color: #b04040 } /* Name.Variable.Instance */
 .highlight .vm { color: #b85820 } /* Name.Variable.Magic */
 .highlight .il { color: #444444 } /* Literal.Number.Integer.Long */
+.highlight .gp { user-select: none } /* Prompt, non-selectable, see #194  */
```

Comment:
Changes to the Pygments theme are in [body.css](https://github.com/pyvec/naucse.python.cz/blob/fbb034221a1a9cc78b30856ff596214a22d41528/naucse/static/css/body.css#L230-L243), to avoid mixing generated and hand-written code.

Reply:
Fixed. Didn't know.